app/src/main/java/io/homeassistant/companion/android/sensors/LastRebootSensorManager.kt

```diff
@@ -49,8 +55,19 @@ class LastRebootSensorManager : SensorManager {
         var local = ""
         var utc = "unavailable"
 
+        val sensorDao = AppDatabase.getInstance(context).sensorDao()
+        val fullSensor = sensorDao.getFull(lastRebootSensor.id)
+        val sensorSetting = sensorDao.getSettings(lastRebootSensor.id)
+        val lastTimeMillis = fullSensor?.attributes?.firstOrNull { it.name == TIME_MILLISECONDS }?.value?.toLongOrNull() ?: 0L
+        val settingOffset = sensorSetting?.firstOrNull { it.name == OFFSET }?.value?.toIntOrNull() ?: 60000

```

Comment:
I'm not sure if offset is the best term to describe this.  It's more of a deadband for the reboot sensor right?

Reply:
Yea that would be a better way to describe it 😄